### DISCUSSION TITLE
有专业领域硕士封面问题

### DISCUSSION THREAD

**Discussion by chwzuo** [tuna/thuthesis#862] [General]
根据2023年3月研究生院发布的研究生学位论文写作指南(202303)，有专业领域的硕士论文封面要加上专业领域

> 1. 专业学位中英文封面调整。
> 对于有“专业领域”的专业学位论文，论文封面需填写专业领域名称。
> 对于无“专业领域”的专业学位论文，论文封面无需填写此信息。
> 2019级及之前工程硕士学位论文填写相应工程领域名称。

<img width="973" alt="image" src="https://user-images.githubusercontent.com/24643217/231990908-685e0927-0460-4d8e-912d-137694193677.png">

主要变动在于：

1. ”申请清华大学工程硕士专业学位论文“变成了”申请清华大学xxx硕士专业学位论文“；
2. ”工程领域“变成了”专业领域“。

现在工程硕士的论文封面似乎还是2019级之前的？

**Comment by zepinglee** [comment]
确实，我改一下。
